index.js

```diff
@@ -82,14 +106,31 @@ function fastifyWebsocket (fastify, opts, next) {
     oldClose.call(this, cb)
   }
 
-  function wsHandle (handle, req, res) {
-    return handle.call(fastify, req[kWs], res)
+  function noHandle (conn, req) {
+    req[kWs].socket.close()
   }
 
-  function noHandle (req, res) {
-    req[kWs].socket.close()
+  function defaultErrorHandler (conn, error) {
```

Comment:
I think the error handler should also include the fastify objects.

I would also use the errorback pattern: `err, conn, req, reply` for example.

Reply:
The error handler is used for errors of both the global and per-route handlers, so it can't take `FastifyRequest`/`FastifyReply` objects.

Would you want it to receive core nodeJS req/res objects?

Reply:
no worries then!